packages/spor-react/src/theme/slot-recipes/switch.ts

```diff
@@ -1,10 +1,8 @@
 import { defineSlotRecipe } from "@chakra-ui/react";
-import { coreBackground, coreBorder } from "../utils/core-utils";
-import { brandBackground } from "../utils/brand-utils";
 import { focusVisibleStyles } from "../utils/focus-utils";
 
 export const switchSlotRecipe = defineSlotRecipe({
-  slots: ["root", "track", "thumb"],
+  slots: ["root", "thumb", "control", "label"],
```

Comment:
vi kan flytte den definisjon inn i anatomy for å være konsekvent med de andre komponenter, det blir mer oversiktlig 😉